src/ProjectTemplates/Microsoft.Extensions.AI.Templates/src/ChatWithCustomData/.template.config/template.json

```diff
@@ -3,9 +3,9 @@
   "author": "Microsoft",
   "classifications": [ "Common", "AI", "Web" ],
   "identity": "Microsoft.Extensions.AI.Templates.Chat.CSharp",
-  "name": "AI Chat with Custom Data",
-  "description": "A project template for creating an AI chat application. It can perform retrieval-augmented generation (RAG) using your own data.",
-  "shortName": "chat",
+  "name": "AI Starter Web App",
+  "description": "A project template for creating an AI chat application, which uses retrieval-augmented generation (RAG) to chat with your own data.",
+  "shortName": "aistarterweb",
```

Comment:
Is this template name and identifier something that came out of a particular decision making process?

I'm not convinced by the name "AI starter web app" because it seems to suggest that AI apps in general are a kind of chat UI, which isn't the case. This project is quite specifically a chat UI.